game-app/game-core/src/main/java/games/strategy/triplea/odds/calculator/AggregateResults.java

```diff
@@ -141,6 +220,26 @@ public double getAverageTuvSwing(
     return count / total;
   }
 
+  /** Returns the standard deviation of the attacking units left if the attacker won. */
+  double getAverageAttackingUnitsLeftWhenAttackerWonStandardDeviation() {
+    if (results.isEmpty()) {
+      return 0.0;
+    }
+    final double mu = getAverageAttackingUnitsLeftWhenAttackerWon();
```

Comment:
`mu` might be hard to follow as a variable name for the casual reader.  `AverageUnits` might be more clear.
Better yet, I think the standard deviation computation can be skipped altogether, this lib looks to have some promising APIs: https://commons.apache.org/proper/commons-math/javadocs/api-2.2/org/apache/commons/math/stat/descriptive/moment/StandardDeviation.html#evaluate(double[],%20double)

Reply:
Sure we can use the lib.  The immediate question is: Should we also replace the average calculation with that lib?
(remark: in our case "average", "expected value", "first momentum" and "mean" all mean the same [pun intended ;-) ]).

Looking at the lib, there is an overhead (object creation, function calls) involved when using it to compute the mean in contrast to calculating it directly.  I assume the AI calls (and thus calculates) the expected value a lot, so this might become a performance regression (though I have not tested it).

Reply:
As of Java 8, invoking functions and creating objects are exceedingly cheap. It can be faster to create an object than it is to invoke a no-op function. Furthermore, those are NS scale operations, there are far greater overheads.

Otherwise, yeah, it's a great idea to also replace the computation for 'average' as well. 'Expected value' is not necessarily average, is it? (We are at risk of starting a mean set of puns)